10-Types-of-contributions.md

```diff
@@ -0,0 +1,233 @@
+
+*Example*
+
+[Insights on social card generation](https://github.com/open-sauced/opengraph/pull/51)
+
+- **Icon Design**: Icons are essential for user interfaces. Graphic designers can create custom icons for software applications or websites, improving the visual appeal and usability of the project.
+
+*Example*
+
+ [Redesign footer](https://github.com/open-sauced/opengraph/pull/51)
+
+ - **Website Design**: Open source projects often have websites to provide information, documentation, and download links. Graphic designers can contribute by designing website layouts, graphics, banners, and other visual elements to make the site more attractive and user-friendly.
+
+*Example*
+
+[Add highlights to the user profile page](https://github.com/open-sauced/app/pull/859)
+
+- **User Interface (UI) Design**: For software projects, UI design is crucial. Designers can create mockups, wireframes, and high-fidelity designs for the user interface, making it more intuitive and visually appealing.
+
+*Example*
+
+[Form page UI implemented
+](https://github.com/zillBoy/pokemon-app/pull/6)
+
+- **User Experience (UX) Design**: UX designers focus on improving the overall user experience by considering how users interact with the software. This includes designing user flows, navigation, and usability testing.
+
+*Example*
+
+ [Improved the user experience of the highlights input form](https://github.com/open-sauced/app/pull/1362)
+
+- **Accessibility Design**: Ensuring that the project's graphical elements are accessible to all users, including those with disabilities, is an important aspect of graphic design. Designers can create accessible color schemes and ensure that visual elements are perceivable and operable.
+
+*Example*
+
+[Redesign footer
+](https://github.com/cesko-digital/pomahejukrajine-web/pull/222)
+
+- **Responsive Design**: Designing visuals that work well on various devices and screen sizes is crucial for modern web applications. Responsive design ensures that the project looks good and functions properly on both desktop and mobile devices
+
+*Example* 
+
+[Enhance feed layout responsiveness 
+](https://github.com/open-sauced/app/pull/1553)
+
+### BLOG WRITING
+
+Blog writing as a type of open-source contribution involves creating and publishing blog posts, articles, or tutorials related to open-source software, projects, technologies, or best practices. Like documentation, you don't necessarily have to know how to code to write these posts. Here are some ways you can use blogging to contribute:
+
+- Reflect on your experiences as a contributor
+- Describe and review the project’s latest updates
+- Mentioning the project when raising awareness about certain topics or in a guide on how to get started in open source
+ 
```

Comment:
[Proofreading] Since your next heading describes the examples, I think it's best to omit this part.
```suggestion
Blog writing as a type of open-source contribution involves creating and publishing blog posts, articles, or tutorials related to open-source software, projects, technologies, or best practices. Like documentation, you don't necessarily have to know how to code to write these posts. 
```